### ISSUE TITLE
Change the size of plot in Matplotlib

### ISSUE THREAD

**Issue by mgorjis** [nteract/archived-desktop-app#2568]
The size of plots sketched with matplotlib can be changed in jupyter notebook via this command:

plt.figure(figsize=(50,30))

However, this command does not have any effect in nteract notebook. Any idea?

**Comment by mgorjis** [comment]
![capture](https://user-images.githubusercontent.com/11069763/36500759-30b72282-1713-11e8-8653-70f1d59caa1a.PNG)

**Comment by a-hurst** [comment]
@mgorjis It's not a very pretty solution, and if possible there should definitely be an easier way to do this in nteract, but after much googling and trial/error I managed to figure out something that works:

```python
import matplotlib

%config InlineBackend.print_figure_kwargs = {'bbox_inches':None}
%config InlineBackend.rc = {'font.size': 10, 'figure.figsize': (10.0, 6.0), 'figure.facecolor': (1, 1, 1, 0), 'figure.subplot.bottom': 0.125, 'figure.edgecolor': (1, 1, 1, 0), 'figure.dpi': 72}
```
, replacing the value of 'figure.figsize' with whatever size you want (I tried breaking down the second dict for better readability, but that doesn't seem to work with ipython directives). The first directive is needed to remove a default max-size limit for inline figures. 

After running those at the top of my notebook, all the `%matplotlib inline` figures display at a much larger and nicer size. Hope this helps!
